packages/e2e-test-utils/src/playwright.ts

```diff
@@ -13,5 +13,5 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/// <reference types="cypress" />
-import './commands';
+
+export * from './playwright/index';
```

Comment:
Do we need this file? Don't we use the dedicated export point anyway

Reply:
Yeah turns our entry point detection in repo-tools didn't like the folder entry, will fix, but later :p